src/mbgl/style/property_parsing.cpp

```diff
@@ -39,7 +39,21 @@ optional<std::string> parseConstant(const char* name, const JSValue& value) {
 
 template <>
 optional<Color> parseConstant(const char* name, const JSValue& value) {
```

Comment:
We should introduce a real class definition for `Color`, rather than using a typedef. I've been meaning to do this for a while. Want to take a stab?